clients/admin-ui/src/features/common/NavBar.tsx

```diff
@@ -35,6 +35,7 @@ const NavLink = ({ title, href, disabled, rightIcon, exact }: NavLinkProps) => {
         rightIcon={rightIcon}
         data-testid={`nav-link-${title}`}
         isActive={isActive}
+        _active={{ bg: undefined }}
```

Comment:
oh yes, I basically wanted to override the themed active color to not have a background color. which I think you're right—is the same as transparent. so I can change to `transparent` since that'll be clearer what's going on 👍